.circleci/config.yml

```diff
@@ -37,43 +32,33 @@ jobs:
             - node_modules
             - yarn.lock
 
-
       - persist_to_workspace:
           root: .
-          paths: .
+          paths: 
+            - .
 
   flow:
     <<: *rn_defaults
     steps:
       - attach_workspace:
           at: ~/project
-      - run:
-          name: Match bitrise's yarn version
-          command: sudo npm install -g yarn@${BITRISE_YARN_VER}
       - run: yarn flow
 
   test:
     <<: *rn_defaults
     steps:
       - attach_workspace:
           at: ~/project
-      - run:
-          name: Match bitrise's yarn version
-          command: sudo npm install -g yarn@${BITRISE_YARN_VER}
-      - run: yarn test
+      - run: yarn test --runInBand
```

Comment:
why run serially, instead of parallel?

Reply:
We are using a free tier in circle-ci, it was the only way to make it run, we were getting too often out-of-memory when running in parallel.